From: Reinhard Tartler <siretart@debian.org>
Date: Fri, 12 Jul 2024 08:36:13 -0400
Subject: Allow netavark to find aardvark-dns by default

---
 src/main.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.rs b/src/main.rs
index 4ee2341..b2aa1fc 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -62,7 +62,7 @@ fn main() {
     let rootless = opts.rootless.unwrap_or(false);
     let aardvark_bin = opts
         .aardvark_binary
-        .unwrap_or_else(|| OsString::from("/usr/libexec/podman/aardvark-dns"));
+        .unwrap_or_else(|| OsString::from("/usr/lib/podman/aardvark-dns"));
     let result = match opts.subcmd {
         SubCommand::Setup(setup) => setup.exec(
             opts.file,
