From: Reinhard Tartler <siretart@tauware.de>
Date: Mon, 17 Feb 2025 13:53:21 -0500
Subject: Relax env_logger dependency

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 816b5cd..410a152 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -33,7 +33,7 @@ deps-serde = ["chrono/serde", "url/serde"]
 [dependencies]
 anyhow = "1.0.93"
 clap = { version = "~4.5.23", features = ["derive", "env"] }
-env_logger = "0.11.6"
+env_logger = { version = ">= 0.11" }
 ipnet = { version = "2.11.0", features = ["serde"] }
 iptables = "0.5.2"
 libc = "0.2.157"
