From: Reinhard Tartler <siretart@tauware.de>
Date: Fri, 30 Aug 2024 21:40:28 -0400
Subject: relax clap dependency

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 410a152..30c0a81 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -32,7 +32,7 @@ deps-serde = ["chrono/serde", "url/serde"]
 
 [dependencies]
 anyhow = "1.0.93"
-clap = { version = "~4.5.23", features = ["derive", "env"] }
+clap = { version = ">= 4.5", features = ["derive", "env"] }
 env_logger = { version = ">= 0.11" }
 ipnet = { version = "2.11.0", features = ["serde"] }
 iptables = "0.5.2"
