From: Reinhard Tartler <siretart@tauware.de>
Date: Fri, 30 Aug 2024 21:42:36 -0400
Subject: Relax nispor dependency

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 57c2d39..25cc115 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -61,7 +61,7 @@ prost = "0.13.4"
 futures-channel = "0.3.31"
 futures-core = "0.3.31"
 futures-util = "0.3.31"
-nispor = "1.2.22"
+nispor = { version = ">= 1.2" }
 tower = { version = "0.5.2", features = ["util"] }
 hyper-util = "0.1.10"
 
